apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-wp-rest-sideload-image-controller.php

```diff
@@ -0,0 +1,286 @@
+<?php
+/**
+ * WP_REST_Sideload_Image_Controller file.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE;
```

Comment:
So here is where you've moved the class into the namespace.

Reply:
Correct